.cases/termsandconditions.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
Getting permission denied on this file. Should it be automatically executable? On my Linux machine it was, could be different on MacOS though..

Reply:
Amazing feedback. I will move those to md files as sh doesnt make sense when we copy commands to execute
We can have template for new verification